.github/CODEOWNERS

```diff
@@ -1,2 +1,154 @@
 # Current reviewers-XXX teams, who review everything for approval.
 * @reviewers-amazon @reviewers-apple @reviewers-comcast @reviewers-google @reviewers-nordic @reviewers-samsung @reviewers-silabs
+
+
+# BUILD system
+
+*.gn @mspang
+/build/ @mspang
+/scripts/build/ @mspang
+
+# Crypto
+# Crypto primitives and high level API, generally on top of mbedtls and openssl
+/src/crypto/* @pan-apple @emargolis
+/src/credentials/ @pan-apple @emargolis
+
+# Transport
+# Send and receive data abstracting away the underlying protocol (UDP, TCP, BLE)
+/src/transport/ @andy31415 @pan-apple
+
+# Session establishment
+# Establish a secure session (encryption keys) from one node to another.
+# Includes CASE and PASE + session tracking and resumptioon
+/src/protocols/secure_channel/ @pan-apple
+
+# Message Layer
+# CRMP and packet headers, generally coupled with SecureSessionManager
+/src/messaging @yufengwangca @kghost @bzbarksy-apple
+/src/transport/raw/MessageHeader.* @yufengwangca @kghost @bzbarksy-apple
+
+# SecureSessionManager
+# Manages per-session encryption keys and new session establishment
+/src/transport/SecureSessionMgr.* @pan-apple
+
+# Exchange management
+# Joins requests and responses or ties a series of messages belonging to the 
+# same protocol interaction
+/src/messaging/ @yufengwangca @bzbarsky-apple
+/src/channel/ @yufengwangca @bzbarsky-apple
+
+# DNS-SD discovery and advertisement
+# Advertise chip devices for commisioning, operational and discover such devices
+/src/lib/mdns/ @andy31415 @gjc13 @damian-nordic
+
+# Commisioning controller/flow
+# Example of a comisioning sequence
+/src/controller/ @pan-apple @erjiaqing
+
+# Operational credentials management
+# Signing certificates, maintaining roots of trust, validation, fabric management
+/src/controller/ @shana-apple @pan-apple
```

Comment:
And `/src/controller/` is present twice, with different owners.

Reply:
@andy31415 

I am quite sure I observed the behavior of this file as the _last_ line to match the path wins rather than gathering all matches.